starboard/android/shared/media_codec_bridge.cc

```diff
@@ -348,6 +359,15 @@ MediaCodecBridge::~MediaCodecBridge() {
     return;
   }
 
+  if (MediaCodecBridgeEradicator::GetInstance()->IsEnabled()) {
+    if (MediaCodecBridgeEradicator::GetInstance()->Eradicate(
```

Comment:
Consider removing the logic of calling stopping and releasing below.  We should always call `Eradicate()` here, and branching on whether it's asynchronous or synchronous inside MediaCodecBridgeEradicator.  So we don't have to duplicate the code for destruction below and inside MediaCodecBridgeEradicator.

Reply:
We could extract the JNI calls into a method, however, I do not feel we should do it now. If the code proves to work, we can do it after it is fully launched.